code/modules/capture_the_flag/ctf_game.dm

```diff
@@ -152,11 +152,12 @@
 	if(player_mob.dna.species.outfit_important_for_life)
 		player_mob.set_species(/datum/species/human)
 
-	player_mob.AddComponent( \
-		/datum/component/temporary_body, \
-		old_mind = new_team_member.mob.mind, \
-		old_body = new_team_member.mob.mind.current, \
-	)
+	if(new_team_member.mob.mind?.current)
+		player_mob.AddComponent( \
+			/datum/component/temporary_body, \
+			old_mind = new_team_member.mob.mind, \
+			old_body = new_team_member.mob.mind.current, \
+		)
```

Comment:
cache `new_team_member.mob.mind` to a proc-local variable so it's easier to read and has a nice cache

```suggestion
	var/datum/mind/new_member_mind = new_team_member.mob.mind
	if(new_member_mind?.current)
		player_mob.AddComponent( \
			/datum/component/temporary_body, \
			old_mind = new_member_mind, \
			old_body = new_member_mind.current, \
		)
```